### PR TITLE
Data collector speedup

### PR DESCRIPTION
**Pull request by cpranav93**
Converted data from torch tensor to dask array for storage with zarr. Speeds up computation significantly from 180s to 50s for 1m agents for 100 ts.

### CODE REVIEW

**Comment by cpranav93** [comment]
Merging this in with gputest for now since master does not have gpu capabilities yet and gputest does not have cpu capabilities. Should fix this asap.

File: dgl_ptm/dgl_ptm/model/data_collection.py [hunk 1]
```diff
@@ -62,7 +63,7 @@ def _edge_property_collector(agent_graph, epath, edata, timestep, format, mode):
                                             ))
             for prop in edata:
                 _check_eprop_in_graph(agent_graph, prop)
-                edge_data_instance = edge_data_instance.assign(property=(['n_edges','time'], agent_graph.edata[prop][:,None].cpu().numpy()))
+                edge_data_instance = edge_data_instance.assign(property=(['n_edges','time'], da.from_array(agent_graph.edata[prop][:,None].cpu().numpy())))
```
Comment (meiertgrootes) [inline dgl_ptm/dgl_ptm/model/data_collection.py:66]: Interesting, so casting the numpy array to a dask array is sufficient to provide the speed up?! nice

Comment (cpranav93) [reply dgl_ptm/dgl_ptm/model/data_collection.py:66]: yeah I was surprised by that too! Also apparently you cannot convert torch tensor to dask array without numpy in the middle.

**Review by meiertgrootes — APPROVED** [review]
looks good to me. o ahead and merge. Interesting how minimal insertion of dask arrays speeds things up!

**Pull request merged**